lib/pymetamap.py

```diff
@@ -9,7 +9,7 @@ def run_command(command_line):
 
 #Remove abnormal characters in the input file, otherwise will lead to system error for metamap
 #TODO: consider more cases
-def format_input(notes_file,outdir):
+def format_input(filename,outdir):
```

Comment:
I didn't delve into the code but I think it's meant to be `filename`. Since `filename` is referenced but never defined, while `notes_file` is defined but never used. Please verify to make sure.